doc/examples/Issue.rst

```diff
@@ -0,0 +1,74 @@
+Issues
+======
+
+Get list of open issues
+-----------------------
+
+.. code-block:: python
+
+    >>> repo = g.get_repo("PyGithub/PyGithub")
```

Comment:
I feel it's more appropriate to put this example under `Repository.rst`, since it's getting repo issues. Other example here is fine. Or simply keep both :)